src/views/projectExplorer/migrateSource.ts

```diff
@@ -144,37 +145,38 @@ export async function migrateSource(iProject: IProject, library: string): Promis
                 }
             }
 
-            // Clean up using Source Orbit
-            const sourceOrbit = extensions.getExtension(`IBM.vscode-sourceorbit`);
-            if (!sourceOrbit) {
-                window.showErrorMessage(l10n.t('Failed to run clean up as Source Orbit extension is not installed'));
-                return migrationResult;
-            }
+            const soEnabled = await sourceOrbitEnabled();
+            const workspaceFolder = workspace.getWorkspaceFolder(migrationConfig.workspaceFolderUri!);
+
+            if (soEnabled) {
+                if (migrationConfig.fixIncludes) {
+                    progress.report({ message: l10n.t('Adjusting include statements to IFS syntax...'), increment: increment });
+                    await commands.executeCommand('vscode-sourceorbit.autoFix', workspaceFolder, 'includes');
+                }
+    
+                if (migrationConfig.generateBob) {
+                    progress.report({ message: l10n.t('Generating Rules.mk for Better Object Builder...'), increment: increment });
+                    await commands.executeCommand('vscode-sourceorbit.generateBuildFile', workspaceFolder, 'bob');
+                }
+    
+                if (migrationConfig.automaticRename) {
```

Comment:
don't we want to do the rename first, before fixing includes or generating build rules?

Reply:
My bad. Didn't mean to re-order